pkg/controller/servicecidrs/servicecidrs_controller.go

```diff
@@ -310,7 +312,7 @@ func (c *Controller) sync(ctx context.Context, key string) error {
 					WithMessage("There are still IPAddresses referencing the ServiceCIDR, please remove them or create a new ServiceCIDR").
 					WithLastTransitionTime(metav1.Now()))
 			svcApply := networkingapiv1beta1apply.ServiceCIDR(cidr.Name).WithStatus(svcApplyStatus)
-			_, err = c.client.NetworkingV1beta1().ServiceCIDRs().ApplyStatus(ctx, svcApply, metav1.ApplyOptions{FieldManager: controllerName, Force: true})
+			_, err = c.client.NetworkingV1beta1().ServiceCIDRs().ApplyStatus(ctx, svcApply, metav1.ApplyOptions{FieldManager: c.name, Force: true})
```

Comment:
- controllerName = "service-cidr-controller" https://github.com/kubernetes/kubernetes/pull/127550/files#diff-37ce781520331481491143bc84a058962599acdc003edf2e905cfa60fb52d8b9L57
- c.name = "service-cidr-controller" https://github.com/carlory/kubernetes/blob/ctrl-name/cmd/kube-controller-manager/names/controller_names.go#L85

FieldManager is not changed.